### PR TITLE
v5.0.0 release post

### PR DESCRIPTION
**Pull request by rvagg**
Binaries are in staging, aiming for a release at 3pm Pacific. Please review this post and let me know if you have any suggestions for changes or additions.

I guess we should ship the new LTS / Stable download button stuff around the same time, plus the ES6 updates? Who can be available to take care of that? I'd rather not have too much on my hands unless it's super-simple.

### CODE REVIEW

File: locale/en/blog/release/v5.0.0.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+date: 2015-10-29T12:33:38.068Z
+version: 5.0.0
+category: release
+title: Node v5.0.0 (Stable)
+slug: node-v5-0-0
+layout: blog-post.hbs
+---
```
Comment (phillipj) [inline locale/en/blog/release/v5.0.0.md:9]: Hmm, `author` field not added, did you use an updated version of `./scripts/release-post.js`?

**Comment by fhemberger** [comment]
LGTM! :+1:

**Comment by fhemberger** [comment]
@rvagg @phillipj How about merging #260, #261 and #268 into the v5.0.0 branch instead of master? So when the release is ready, we just need to merge a single branch.

**Comment by evanlucas** [comment]
That seems like an ideal way to do it

**Comment by phillipj** [comment]
Other than the missing `author: Rod Vagg` field it LGTM!

@fhemberger IMO LTS/Stable buttons in #260 and #261 could be merged ASAP. Both has been written with this in mind, too reduce big bang release of nodejs.org as soon as v5 gets released.

**Comment by fhemberger** [comment]
Added ES6 updates here, changes for download page and index are already merged.

File: locale/en/blog/release/v5.0.0.md (hunk 1 above)
Comment (mikeal) [inline locale/en/blog/release/v5.0.0.md:9]: author shouldn't be strictly required.

**Comment by rvagg** [comment]
done, thanks folks

**Pull request merged**